realm/realm-library/src/main/java/io/realm/mongodb/sync/Subscription.java

```diff
@@ -0,0 +1,101 @@
+package io.realm.mongodb.sync;
+
+import java.util.Date;
+
+import javax.annotation.Nullable;
+
+import io.realm.RealmQuery;
+import io.realm.annotations.Beta;
+import io.realm.internal.Util;
+import io.realm.internal.annotations.ObjectServer;
+import io.realm.internal.UnmanagedSubscription;
+
+/**
+ * A subscription defines a specific server query and its metadata. The result of this query
+ * is continuously being synchronized with the device as long as the subscription is part of a
+ * {@link SubscriptionSet} with a state of {@link SubscriptionSet.State#COMPLETE}.
+ *
+ * Subscriptions are immutable once created, but they can be updated by using a
+ * {@link MutableSubscriptionSet#addOrUpdate(Subscription)}.
+ */
+@ObjectServer
+@Beta
+public interface Subscription {
+
+    /**
+     * Create an unmanaged named subscription for a flexible sync enabled Realm.
+     * The subscription will not take effect until it has been store using either
```

Comment:
```suggestion
     * The subscription will not take effect until it has been stored using either
```

There are a few more occurrences of this typo down here